### PR TITLE
Update to Who We Are

### PR DESCRIPTION
**Pull request by tyler-patterson-chr**
First thoughts on digital paper...

### CODE REVIEW

**Comment by tyler-patterson-chr** [comment]
@kyle-spitzer-chr is helping out with this. we are going to be seeking department input on this topic to help build it out. More to come.

File: who-we-are/index.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ image:
...
+_We’re the original 3PL.  We believe in accelerating global trade to seamlessly deliver the products and goods that drive the world’s economy. With more than a century of industry experience, we know what it takes to manage rapid growth, embrace change, and to stay competitive in the marketplace. Today, we’re one of the world’s largest third party logistic providers—delivering innovative supply chain solutions to more than 113,000 customers across the world. Whether you need truckload services, ocean or air shipping, supply chain consulting, or more—we use the strengths of our knowledgeable people, proven processes, and global technology to provide unmatched connectivity, capabilities, and experience._
 
-A description of who CHRITters are.
+When it comes to CHR IT, we live and breathe our role in those words to deliver on our vision of accelerating commerce through the world’s most powerful supply chain platform. However, the role we play as CHR IT is really the outcome of all of our CHR IT-er's, or CHRITers.
```
Comment (brandy-churchill-chr) [inline who-we-are/index.md]: CHRITters

File: who-we-are/index.md [hunk 2]
```diff
@@ -10,6 +10,16 @@ image:
...
+
+At our core, CHRITers are:
```
Comment (brandy-churchill-chr) [inline who-we-are/index.md]: CHRITters

File: who-we-are/index.md (hunk 1 above)
Comment (brandy-churchill-chr) [inline who-we-are/index.md]: The role we play as CHR IT...

instead of

However, the role we play...

**Comment by abby-d-chr** [comment]
Below are suggested edits and stuff I made up. Feel free to tweak and revise as desired. (This is not a pull request be cause I don't have access to create one.)
---
We are C.H. Robinson IT (CHR IT).  We work to deliver our vision of accelerating commerce through the world’s most powerful supply chain platform.
 
The role of CHR IT is determined by each of our dedicated CHR IT-ers, or CHRITers*.
 
At our core, CHRITers are
1. [I assume you already have descriptors you'd like to go here]
2. ?
3. ?

We want to share our experiences, challenges, and triumphs to help make IT a better place worldwide.
 
* Pronounced _critters_

**Pull request merged**